stubs/ArrayObject.stub

```diff
@@ -105,3 +105,66 @@ class SplFixedArray implements Iterator, ArrayAccess, Countable
 	 */
 	public function toArray(): array { }
 }
+
+/**
+ * @template TKey of object
```

Comment:
You understand how SplObjectStorage works wrong. The user cannot influence the iterable key, it's always int: https://3v4l.org/J4M0N

Reply:
Ok, but the ArrayAccess Key is an objet: https://3v4l.org/DsUbK

So it would be
```
@template-implements Iterator<int, TKey>
@template-implements ArrayAccess<TKey, TValue>
```
?

Reply:
Yeah but it'd make more sense to rename them to `TObject` and `TData`.

Reply:
Done ! I have only 3 build failures. It's for php 8 so I assume it's normal.